GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/CarStorePageController.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CarStorePageController : MonoBehaviour
+{
+    public GameObject itemSedan;
+    public GameObject itemTruck;
+    public GameObject itemJeep;
+    public GameObject itemKart;
+    public GameObject confirmPanel;
+    public Text confirmText;
+
+    private int _priceInCoin;
+    private string _itemNameToPurchase;
```

Comment:
Can you define an object for the car? The object can contain the name, price, coin vs $. Then you can pass the object around, instead of setting these two variables and passing them around.